code/game/objects/items/stacks/medical.dm

```diff
@@ -197,9 +197,14 @@
 
 			H.UpdateDamageIcon()
 			use(1)
+			affecting.status |= ORGAN_SALVED
+			addtimer(CALLBACK(affecting, /obj/item/organ/external/.proc/deointmenterized), 200 SECONDS)
 		else
 			to_chat(user, "<span class='warning'>[affecting] is cut open, you'll need more than some ointment!</span>")
 
+/obj/item/organ/external/proc/deointmenterized()
```

Comment:
```suggestion
/obj/item/organ/external/proc/remove_ointment()
```
way easier to read

Reply:
you're no fun, Farie

Reply:
`remove_flattest()`

Reply:
We got Oxford on this. `deointmenterized` has been officially defined in all dictionaries. 😎

Reply:
As the resident grammar reviewer, deointmenterized should stay